AirCasting/TabBar/TabBarSelector.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+import Combine
+
+class TabBarSelector: ObservableObject {
+    enum Tab {
+        case dashboard
+        case createSession
+        case settings
+    }
+    
+    @Published private(set) var selection = Tab.dashboard {
+        didSet {
+            updateSelectedIcon()
+        }
+    }
+    
+    var dashboardSelectionNotifier = PassthroughSubject<Void, Never>()
+    private var tabBarIcons: [Tab: TabIcon] = [
+        .dashboard: TabIcon(state: .selected,
+                            selectedString: Strings.MainTabBarView.homeBlueIcon,
+                            unselectedString: Strings.MainTabBarView.homeIcon),
+        .settings: TabIcon(selectedString: Strings.MainTabBarView.settingsBlueIcon,
+                           unselectedString: Strings.MainTabBarView.settingsIcon),
+        .createSession: TabIcon(selectedString: Strings.MainTabBarView.plusBlueIcon,
+                                unselectedString: Strings.MainTabBarView.plusIcon)
+    ]
+    
+    func updateSelection(to newSelection: Tab) {
+        selection = newSelection
+        
+        if newSelection == .dashboard {
+            dashboardSelectionNotifier.send(())
+        }
+    }
```

Comment:
I think you can remove the empty space 🤔
And maybe, just maybe you could rename it to just:
 ```
 func select(tab: Tab) {
        selection = newSelection

        if newSelection == .dashboard {
            dashboardSelectionNotifier.send(())
        }
    }
```
or 
 ```
 func update(to: Tab) {
        selection = newSelection

        if newSelection == .dashboard {
            dashboardSelectionNotifier.send(())
        }
    }
```
Because you always use it on `tabSelection` so it's  making it shorter and less repetitive :)